articles/appliance/update.md

```diff
@@ -1,21 +1,23 @@
-# Updating an Auth0 Cluster
+# Update an Auth0 Cluster
 
-Download the package provided by Auth0 to your local machine and verify its integrity by comparing the checksum provided by Auth0 with a computed one as explained [here](/checksum).
+To update an Auth0 cluster, follow these steps:
 
-Open the dashboard and navigate to **Configuration**:
+1. Download the package provided by Auth0 to your local machine.
+2. Verify its integrity by comparing the checksum provided by Auth0 with a computed one as explained in [Verify Update Package Integrity](/checksum).
+3. Open the dashboard and navigate to **Configuration**:
 
-![go to the configuration menu](/media/articles/appliance/update/offline-update-01.png)
+ ![go to the configuration menu](/media/articles/appliance/update/offline-update-01.png)
 
-Go to the **Offline Update** section and click on **Update from Package** and select the package from the file system:
+4. Go to the **Offline Update** section and click on *Update from Package* and select the downloaded package from the file system:
```

Comment:
For consistency, I've been leaving references to menu items in bold, but placing references to buttons and text items in italics.